tests/container_tests/test_zip_container.py

```diff
@@ -84,6 +108,7 @@ def test_makedirs(self):
             self.assertRaises(io.UnsupportedOperation,
                               handler.makedirs, "test/test")
 
+    @pytest.mark.timeout(1)
```

Comment:
Why this timeout is introduced? Introducing non-determinism makes tests very flaky thus must be carefully designed and intended.

Reply:
As I stated in the comment message, I want to make sure the pickle problem is solved in pytest

Reply:
The timeout value need to be chosen again probably

Reply:
Move this to another PR